api.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from flask_script import Manager
+from flask_script import Server
```

Comment:
We should review whether using flask script is needed, https://flask-script.readthedocs.io/en/latest/ recommends looking at flasks built in cli tool.